pkgs/by-name/oi/oil/package.nix

```diff
@@ -5,13 +5,13 @@ let
     name = "readline-all"; paths = [ readline readline.dev ];
   };
 in
-stdenv.mkDerivation rec {
+lib.trivial.warn "'oil' is deprecated in favor of the much faster C++ compiled binary 'oils-for-unix'. It will be removed soon. See also https://github.com/oils-for-unix/oils/wiki/Oils-Deployments" stdenv.mkDerivation rec {
```

Comment:
This warning will show up in every `nix-env -q` and `nix search`, so we shouldn't have it as it becomes spam my if many packages do that. We can add this to `aliases.nix` if we actually remove it.

Reply:
Thanks a lot for the review & pointer!
Since both packages give the same output - except one is transpiled+compiled and one is only interpreted - I just deprecated `oil` now completely. It's also what upstream recommends.